apps/calendar/js/object.js

```diff
@@ -12,6 +12,23 @@ exports.filter = function(obj, fn, thisArg) {
   return results;
 };
 
+exports.find = function(obj, fn, thisArg) {
```

Comment:
are you using this somewhere? couldn't find any calls to it besides the unit tests.